masterlist.yaml

```diff
@@ -4956,16 +4693,7 @@ plugins:
     inc:
       - 'TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp'
     msg:
-      - <<: *hasWildEdits
-        subs:
-          - |+
-
-              #### Remove the following records with xEdit
```

Comment:
If this information isn't going to be included in a message, it should probbaly be made available somewhere else and linked to. Unless it's inaccurate?

Reply:
I haven't had a chance to check if it's still relevant. I could make a post in AFK forums for wild edit cleaning guides, which would be separate to the LOOT page. I'm not sure where else would be appropriate to host them.

Reply:
Some plugins have links to a Creation Kit wiki page, IIRC, you could put the info there.

Reply:
Creation Kit wiki page is good, I'll set up an account and add the guides there.

Reply:
Okay tried the wiki and the captcha makes it a pain.
I could use a github page with markdown.
https://cptmcsplody.github.io/guides/

Reply:
I've added the guides here https://cptmcsplody.github.io/Guides/ for the moment, at least before I translate them to the Wiki format.

Reply:
I've added the guides here https://cptmcsplody.github.io/X-Guides/

Reply:
Resolved in https://github.com/loot/skyrimse/commit/2acab29df606cad6755c3d5eb565de684ee322d5